docs/release/kubernetes-upgrade.md

```diff
@@ -143,7 +143,8 @@ go mod tidy
 Ensure to update the Go version in both the Dockerfile and GitHub workflows, if required.
 
 ```sh
-sed -i'' “s/<OLD_GO_VERSION>/<NEW_GO_VERSION>/g” Dockerfile.* .github/workflows/integration.yaml .github/workflows/unitcoverage.yaml
+export OLD_GO_VERSION=<old-go-version>
+sed -i'' "s/$OLD_GO_VERSION/$GOVERSION/g" Dockerfile.* .github/workflows/integration.yaml .github/workflows/unitcoverage.yaml
```

Comment:
It would work, but the double quotes would work as well. The problem with the old one is that it was using some stylized double quotes and not the default one.